dev/com.ibm.ws.microprofile.config.1.2_fat/fat/src/com/ibm/ws/microprofile/config12/test/FATSuite.java

```diff
@@ -16,9 +16,7 @@
 
 @RunWith(Suite.class)
 @SuiteClasses({
-                ConverterPriorityTest.class,
-                TypeConverterTest.class,
-                ImplicitConverterTest.class,
+                Config12ConverterTests.class
```

Comment:
For consistency with other FatSuite classes, this could be commented with `//LITE`